### ISSUE TITLE
Motherboard test: PSG integration

### ISSUE THREAD

**Issue by apoloval** [apoloval/artemisa#93]
## Description

Connect the PSG and run a Basic program that reproduces a short theme. 

## Scenario

- [x] PSG reproduces a short theme
GIVEN the following Basic program
WHEN program is ran 
THEN the theme is reproduced as [in this video](https://www.youtube.com/watch?v=N4DSvK3qS94). 

<img width="474" alt="Captura de pantalla 2019-06-25 a las 20 36 18" src="https://user-images.githubusercontent.com/738507/60123973-ebdd3500-9788-11e9-8b7a-a157bf14e779.png">

## Connections

- [x] Those from #91 
- [x] U23, the PSG
- [x] R7, R8 and R9, the resistors from PSG output channels and JP keyboard mode

<img width="710" alt="Captura de pantalla 2019-06-25 a las 20 24 36" src="https://user-images.githubusercontent.com/738507/60123479-d61b4000-9787-11e9-8d31-94fdff1cdd79.png">

- [x] C29 and C30, the DC removing capacitors for sound channels
- [x] U22 and U24, the 74157s demuxers for joystick ports
- [x] RN3 and RN4, pullup resistors for joystick ports

**Comment by apoloval** [comment]
Test passed ([video](https://twitter.com/ArtemisaMSX/status/1143782339857846272?s=20)), with some issues:

- The introduction of the PSG added some additional mA more of power consumption. This caused the power supply to provide less voltage than expected in the input (4.6v), and some devices start to malfunction. I had to regulate the voltage in the power source to 5.3v to have 4.9v in the board. After that, these issues disappeared. I don't know yet if this is caused by the power supply (I hope). 

- The system hangs after some commands in Basic. For instance, if we: 
  - Introduce line 10 as in the program above
  - Introduce `20 PLAY A$`
  - Execute `RUN`
  - The program works and it sounds as expected. 
  - Introduce `30 END`
  - The system hangs

The good news are that this is caused by the MSX System ROM I am using. I can reproduce it in OpenMSX, exactly the same steps and the same results. So it is a software issue, the hardware seems to be okay. 

I tried with a different system ROM by enabling the `NDEVFIX`. No luck, same results in the emulator. I still have to figure out why the MSX system is not working.
